### ISSUE TITLE
Training with data other than coco

### ISSUE THREAD

**Issue by esgnn** [stefanopini/simple-HRNet#43]
Hi,

Thank you for the easier to use implementation.
I am currently trying to train a pre-trained weight with leeds extended sports data. I have converted keypoint annotations to COCO format, but head points do not match, so I assume they are not available. Also leeds has points either visible or not, so no visibile==1 (COCO). 
Long story short, training actually declined the accuracy. So, do you think lack of data compatibility is an issue?

**Comment by stefanopini** [comment]
Hi!
Yes, it could be.

Some thoughts:
- Joints order of COCO and Leeds Sports Pose Extended Training Dataset is different so double check that you have correctly re-ordered Leeds joints to match [COCO joint order](https://github.com/stefanopini/simple-HRNet/blob/master/misc/visualization.py#L11-L30)
- Joints of Leeds Sports Pose Extended Training Dataset seems to be more similar to [MPII Human Pose joints](https://github.com/stefanopini/simple-HRNet/blob/master/misc/visualization.py#L41-L59). Starting from MPII pre-trained weights may result in higher performance.
- Use a low learning rate and try with different learning rates because it seems to have a high impact on training when starting from pre-trained weights (see issues #26 and #29)

**Comment by esgnn** [comment]
- Thank you for taking your time. Sorry, missed #26 and #29. Actually decreasing learning rate solved acute drop. 
- I'll give MPII a try.
- Do you think missing keypoints (eyes, ears, nose) could be annotated by predicting those images with a pretrained model? Is there a way to recover whether model predicts a point's visibility?

Thanks again

**Comment by stefanopini** [comment]
> Do you think missing keypoints (eyes, ears, nose) could be annotated by predicting those images with a pretrained model?

Yes.

> Is there a way to recover whether model predicts a point's visibility?

As far as I know, no because the model doesn't predict that. Actually, the point confidence is usually used to filter the keypoints.
